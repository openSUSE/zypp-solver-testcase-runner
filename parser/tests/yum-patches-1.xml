<?xml version="1.0" encoding="UTF-8"?>
<patches xmlns="http://novell.com/package/metadata/suse/patches">
  <patch id="12345">
    <checksum type="sha">e7ba668ab0d49486f0a3e7a0b168406a88a46ed9</checksum>
    <location href="repodata/patch.xml"/>
  </patch>
  <patch id="54321">
    <checksum type="md5">e7ba668ab0d49486f0a3e7a0b168406a88a46ed9</checksum>
    <location href="repodata/patch2.xml"/>
  </patch>
</patches>
