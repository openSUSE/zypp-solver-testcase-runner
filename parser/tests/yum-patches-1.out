-------------------------------------------------
Patches Entry Data: 
  patch ID: 12345
  location: repodata/patch.xml
  checksumType: 'sha'
  checksum: 'e7ba668ab0d49486f0a3e7a0b168406a88a46ed9'
-------------------------------------------------
Patches Entry Data: 
  patch ID: 54321
  location: repodata/patch2.xml
  checksumType: 'md5'
  checksum: 'e7ba668ab0d49486f0a3e7a0b168406a88a46ed9'
Exit Code: 0
